tests/components/homewizard/test_sensor.py

```diff
@@ -897,6 +909,7 @@ async def test_external_sensors_unreachable(
                 "sensor.device_voltage_swells_detected_phase_2",
                 "sensor.device_voltage_swells_detected_phase_3",
                 "sensor.device_water_usage",
+                "sensor.device_wi_fi_strength",
```

Comment:
Why did you also change the wifi_strength in this test file?

Reply:
Sorry, this is indeed a bit unrelated.

The reason is that is should have been there in the first place, it was missing and spotted it during the development of this PR. Same for the change below.

Do you want me to remove those?